### ISSUE TITLE
Fails to fetch when using all regions

### ISSUE THREAD

**Issue by antonbabenko** [cloudquery/cloudquery#94]
Following getting started for the first time (no `region` is specified in `config.yml`):

```
$ cloudquery fetch --dsn "host=localhost user=postgres password=pass DB.name=postgres port=5432"
7:33PM INF logging configured consoleLog=true fileLogging=true fileName=cloudquery.log jsonLogOutput=false logDirectory=. maxAgeInDays=3 maxBackups=3 maxSizeMB=30 verbose=false
7:33PM INF requesting provider initialize provider=aws version=latest
{"@level":"debug","@message":"creating table if not exists","@timestamp":"2021-05-13T19:33:32.213624+02:00","table":"aws_autoscaling_launch_configurations"}
...
7:33PM INF requesting provider fetch provider=aws version=latest
{"@level":"info","@message":"No regions specified in config.yml. Assuming all 22 regions","@timestamp":"2021-05-13T19:33:33.756055+02:00"}
{"@level":"info","@message":"fetched successfully","@timestamp":"2021-05-13T19:33:35.566516+02:00","Region":"us-east-1","account_id":"835367859851","count":0,"table":"aws_ec2_instances"}
...
Error: rpc error: code = Unknown desc = operation error EC2: DescribeNetworkAcls, retry quota exceeded, 0 available, 5 requested
```

Works well when limiting to one region in `config.yml`:
```
providers:
    - name: aws
      version: latest
      regions:
        - eu-west-1
      resources:
        # ...
```

**Comment by yevgenypats** [comment]
Thanks for reporting! Can you try adding `max_retries: 20` to the `config.yml` ?

**Comment by antonbabenko** [comment]
Does not seem to help. Different API call but the same error message:
> Error: rpc error: code = Unknown desc = operation error EC2: DescribeInternetGateways, retry quota exceeded, 0 available, 5 requested

**Comment by roneli** [comment]
Should be fixed in CloudQuery v0.13.2 and AWS v0.4.3 (https://github.com/cloudquery/cq-provider-aws/pull/70). 
Note: v0.13 has changed to use a hcl config instead of a yml. It is very similar. You can generate one with ./cloudquery init aws

**Comment by antonbabenko** [comment]
Closing as fixed.
